### PR TITLE
Basic HTML-Structure

### PR DESCRIPTION
**Pull request by bitPogo**

### CODE REVIEW

**Comment by wiese** [comment]
@bitPogo I took the liberty to rebase this on top of master for your convenience.

File: src/components/TermBox.vue [hunk 1]
```diff
@@ -1,37 +1,79 @@
 <template>
```
Comment (wiese) [inline src/components/TermBox.vue]: This lacks all kinds of test.

File: src/components/TermBox.vue [hunk 2]
```diff
@@ -1,37 +1,79 @@
...
 		} ),
 		...mapGetters( NS_LANGUAGE, {
 			getLanguageTranslation: 'getTranslationByCode',
 		} ),
 	},
 } )
+
 export default class TermBox extends ( Vue as VueConstructor<TermboxBindings> ) {
+	get label(): Term {
+		return this.entityLabel( this.primaryLanguage );
```
[the remark this answers is not in the document]
Comment (bitPogo) [reply src/components/TermBox.vue]: Done. In any case, I think that the components should get a empty string/list from the store and should not aware of the internal structure of the JSON blob, which also includes nested stuff. (just that I put that some where)

File: src/components/TermBox.vue [hunk 3]
```diff
@@ -1,37 +1,79 @@
 <template>
-	<div class="termbox">
-		<div class="primary-language">{{primaryLanguageName}}</div>
+	<div id="wikibase-termbox-primaryLanguage">
```
[the remark this answers is not in the document]
Comment (bitPogo) [reply src/components/TermBox.vue]: done

File: src/components/TermBox.vue (hunk 1 above)
Comment (bitPogo) [reply src/components/TermBox.vue]: done

File: tests/unit/components/TermBox.spec.ts [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+		},
+	} as LanguageTranslations,
+);
+
+describe( 'TermBox.vue', () => {
+	it( 'renders the language label', () => {
+		store.commit( `${NS_USER}/${LANGUAGE_INIT}`, 'de' );
+
+		const wrapper = shallowMount( TermBox, { store, localVue } );
+		expect( wrapper.text() ).toContain( langDe );
```
Comment (wiese) [inline tests/unit/components/TermBox.spec.ts]:
```suggestion
		expect( wrapper.find( '.wikibase-termbox-language' ).text() ).toBe( langDe );
```

would be my preferred approach.
`.text()` of the complex wrapper is really just a bunch of innerHtml and asserting against this is a game of chance.

File: src/components/TermBox.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (wiese) [reply src/components/TermBox.vue]: Went with printing question marks ("?") for now.

**Comment by wiese** [comment]
Squashed and rebased this.

**Pull request merged**